core/src/components/select/select.ionic.scss

```diff
@@ -11,6 +11,7 @@
   // ios and md. This will allow us to have a single color map.
   --border-color: #{globals.current-color(neutral)};
   --border-width: #{globals.$ion-border-size-025};
+  --focus-ring-color: #{globals.$ion-border-focus-default};
```

Comment:
There's already a focus variable: `--highlight-color-focused`. I recommend reviewing the [docs](https://ionicframework.com/docs/api/select#css-custom-properties).
- https://github.com/ionic-team/ionic-framework/blob/ef9a8ef550b771c5c8154f1db95343e85396d5e0/core/src/components/select/select.common.scss#L37

Reply:
Ah silly me, I saw that and my brain instantly thought about text-highlight for some reason :). Will make the needed changes!

Reply:
Implemented the changes! Added only for the outline, as its the only type used on Ionic theme.

Also took the opportunity to fix the border color token, thats why we have so many snapshots updated!